lego/apps/companies/views.py

```diff
@@ -46,8 +46,10 @@
 
 class AdminCompanyViewSet(AllowedPermissionsMixin, viewsets.ModelViewSet):
     queryset = Company.objects.all().prefetch_related("semester_statuses", "files")
-    pagination_class = None
     permission_handler = CompanyAdminPermissionHandler()
+    filter_backends = [filters.OrderingFilter]
+    ordering_fields = ["name", "created_at"]
+    ordering = "name"
```

Comment:
Can we move this to `lego/apps/companies/filters.py`, like we do for the other view sets